modules/dbfs/multi.go

```diff
@@ -0,0 +1,226 @@
+package dbfs
+
+import (
+	"strconv"
+
+	"fmt"
+
+	"github.com/CodeCollaborate/Server/modules/patching"
+	"github.com/CodeCollaborate/Server/utils"
+)
+
+// TODO (wongb): Move this to config
```

Comment:
I still want to move this to config eventually; depending on the servers the binary runs on, we may or may not want to increase/decrease the history.

I wasn't planning to put this in this specific PR, as I wanted to see how much the server affected these values.